src/tv2_offtube_showstyle/helpers/config.ts

```diff
@@ -36,20 +36,21 @@ export interface DVEConfigInput {
 
 export interface OfftubeShowStyleConfig extends TV2ShowstyleBlueprintConfigBase {
 	WipesConfig: TableConfigItemValue
+	GraphicsSetups: TableConfigGraphicsSetup[]
 }
 
-function findGraphicsSetup(
-	_context: ICommonContext,
-	_config: TV2ShowstyleBlueprintConfigBase
-): TableConfigGraphicsSetup {
-	// just for type compatibility, not really supported in offtube
-	return {
-		Name: '',
-		VcpConcept: '',
-		OvlShowId: '',
-		FullShowId: '',
-		DveLayoutFolder: ''
+function findGraphicsSetup(context: ICommonContext, config: OfftubeShowStyleConfig): TableConfigGraphicsSetup {
```

Comment:
This method seems very similar to the `findGraphicsSetup` for Galleries. It seems it is the type of the parameter config that differs. Couldn't we extract the two methods so we don't have this logic twice? Or does `OfftubeShowStyleConfig` not have anything in commen with the gallery version?

Reply:
I replaced it with a generic implementation in tv2-common